### PR TITLE
add mergevec.py to simplify the workflow

### PR DESCRIPTION
**Pull request by fabianbormann**
In newer versions of opencv (3.1) it's very difficult to compile mergevec.cpp, because the needed files are (re)moved or renamed. Therefore it's easier to use the python version (https://github.com/wulfebw/mergevec).

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,4 +1,5 @@
 `createsamples.pl` and `mergevec.cpp`: Copyright (c) 2008, Naotoshi Seo
+`mergevec.py`: Copyright (c) 2014, Blake Wulfe
```
Comment (mrnugget) [inline LICENSE]: Could you move this line two down, so the "From: ..." is below "createsamples.pl ..." line? Maybe you could also add a URL pointing to the mergevec.py repository

**Comment by mrnugget** [comment]
Wow! Thanks! That looks awesome 👍  👍  👍 

Does the change have any effect on compatibility with OpenCV 2.x?

**Comment by fabianbormann** [comment]
Hi,
there is no effect. The only difference is that mergevec.py runs independent from OpenCV so you don't need a specific OpenCV version. :)

Cheers,
Fabian

**Comment by mrnugget** [comment]
Thanks a lot for this, @fabianbormann! If you have more ideas as to how to make this repo cleaner and maybe make it compatible with newer versions of OpenCV, I could give you commit access. Just say the word 😃

**Pull request merged**